tests/plugins/test_rss_validation.py

```
@@ -0,0 +1,152 @@
+"""JTN-380: URL validation for the RSS plugin's feed URL field.
+
+Prior behavior silently accepted any non-empty string (e.g.
+``definitely-not-a-feed-url``), surfacing the failure later when
+``generate_image`` tried to fetch the feed. We now reject non-http(s) URLs at
+save time and render the input as ``type="url"`` for client-side feedback.
+"""
+
+import pytest
+
+
+def _make_plugin():
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_make_plugin`

(ANN202)